db/migrate/20200814134116_report_state_stored_as_string.rb

```diff
@@ -0,0 +1,6 @@
+class ReportStateStoredAsString < ActiveRecord::Migration[6.0]
+  def change
+    change_column :reports, :state, :string, default: "inactive"
+    add_index :reports, :state
```

Comment:
Will reports be queries purely on `state`? Or are they view in the context of a fund so that a compound index on `[:fund_id, :state]` would make more sense?

Reply:
I can't think of any places right now where we would want reports by fund - but I can easily imagine the time when we do - can we add the compound index at that point or is it something we need to decide upfront?

Reply:
If we don't need that functionality right now that's fine. I was just checking whether for BEIS users we list reports out by fund, or just list all submitted reports as a single list.